app/views/patients/show.html.erb

```diff
@@ -97,7 +97,7 @@
         </tr>
       </thead>
       <tbody>
-        <% @patient.assessments.order('created_at').each do |assessment| -%>
+        <% if @assessments.length > 0 %>
```

Comment:
We need to be looping here still, looks like a line was maybe deleted?

Reply:
also must have missed this in a rebase - fixed now